spec/router_spec.js

```diff
@@ -321,6 +321,74 @@ describe('Router', function() {
         router.params({a: undefined, b: '22', c: false, d: null, e: 0});
         expect(router.params()).toEqual({b: '22', e: 0});
       });
+
+
+      describe('with replaceState', function() {
+        beforeEach(function() {
+          router._timer = undefined;
```

Comment:
@burrows I am concerned about this line. I am using ```jasmine.clock()```  for the first time in these specs. There was flakiness because of the setTimeouts in the specs and the setTimeout on line 365 in router.js. Is there a better way to handle this?